docs/theories/theories.rst

```diff
@@ -21,7 +21,6 @@ Standardized theories
    Theory of integer arithmetic <https://smtlib.cs.uiowa.edu/theories-Ints.shtml>
    Theory of real arithmetic <https://smtlib.cs.uiowa.edu/theories-Reals.shtml>
    Theory of mixed-integer arithmetic <https://smtlib.cs.uiowa.edu/theories-Reals_Ints.shtml>
-   Theory of strings <https://smtlib.cs.uiowa.edu/theories-UnicodeStrings.shtml>
```

Comment:
I think it makes sense to still list this here.  The other extensions of theories (e.g. transcedentals extend arithmetic), we keep the link to the standard theory here.

Reply:
Done.